### PR TITLE
Bump zwave-js to 8.7.5 and bump zwave-js-server to 1.10.8

### PR DESCRIPTION
**Pull request by raman325**
We need to do this to release the bug that was fixed in https://github.com/zwave-js/zwave-js-server/pull/427 because we want to bump the addon to a version that has the `Driver_Failed` feature. We should not wait for #432 to merge this

### CODE REVIEW

File: src/lib/controller/message_handler.ts [hunk 1]
```diff
@@ -191,55 +191,57 @@ function processInclusionOptions(
...
-            ) {
-              delete clientsController.grantSecurityClassesPromise;
-            }
-          });
-          client.sendEvent({
-            source: "controller",
-            event: "grant security classes",
-            requested: requested as any,
-          });
+      if (!("provisioning" in options)) {
```
Comment (MartinHjelmare) [inline src/lib/controller/message_handler.ts:194]: Are these changes intended?

Comment (raman325) [reply src/lib/controller/message_handler.ts:194]: yes, this is required, otherwise the package won't build due to an addiitonal options schema that was added

**Review by balloob — APPROVED** [review]
